templates/controllers/wizard/fileUpload/form/supplementaryFileMetadataForm.tpl

```diff
@@ -0,0 +1,62 @@
+{**
+ * templates/controllers/wizard/fileUpload/form/supplementaryFileMetadataForm.tpl
+ *
+ * Copyright (c) 2014-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Supplementary file metadata form.
+ *
+ * Parameters:
+ *  $submissionFile: The submission artwork file.
+ *  $stageId: The workflow stage id from which the upload
+ *   wizard was called.
+ *  $showButtons: True iff form buttons should be presented.
+ *}
+<script type="text/javascript">
+	$(function() {ldelim}
+		// Attach the form handler.
+		$('#metadataForm').pkpHandler('$.pkp.controllers.form.AjaxFormHandler');
+		$('input[id^=\'dateCreated\']').datepicker();
```

Comment:
I disagree -- the amount of overhead doesn't justify this, in my opinion. One thing we could do is flag date picker controls e.g. by class, then add them to SiteHandler or something so that they're automatically initialized.